src/render/attrs/index.js

```diff
@@ -26,15 +31,33 @@ function renderAttrs (attrs) {
       !attrsNotToRender[attrKey]
     ) {
       let attrVal = attrs[attrKey];
+      const explicitBooleanValue = attrsWithExplicitBoolValue.includes(attrKey);
```

Comment:
Could `attrsWithExplicitBoolValue` be an object like `attrsNotToRender` to avoid the array search?